tests/test_transpile.py

```diff
@@ -845,7 +845,6 @@ def test_command_identity(self):
             "ALTER TABLE table1 RENAME COLUMN c1 TO c2, c2 TO c3",
             "ALTER TABLE table1 RENAME COLUMN c1 c2",
             "ALTER TYPE electronic_mail RENAME TO email",
-            "ALTER VIEW foo ALTER COLUMN bla SET DEFAULT 'NOT SET'",
```

Comment:
We shouldn't get rid of this test. Let's make sure this is still supported (either parses into an alter expression or into a command), otherwise this PR will introduce a regression.

Reply:
This test is for [unsupported syntax](https://github.com/tobymao/sqlglot/pull/3873/files/08706b9b4bab7fb35e520dcb732a70ab6100501a#diff-ddc45c8cfc37c82c0e36c52bcd13d8de90b866af0a3150bdefc157729db6d63dR867). I will move it to another test.

Reply:
Actually I can't find any SQL dialect supports "ALTER VIEW ... ALTER COLUMN ..."
Only T-SQL supports something [similar](https://learn.microsoft.com/en-us/sql/t-sql/statements/alter-view-transact-sql?view=sql-server-ver16#syntax) but requires two statements like below:
```
ALTER VIEW view_name
ALTER COLUMN column_name datatype [NULL | NOT NULL];

```